src/Nri/Ui/ClickableText/V1.elm

```diff
@@ -0,0 +1,163 @@
+module Nri.Ui.ClickableText.V1 exposing
+    ( button, link
+    , Size(..)
+    )
+
+{-|
+
+
+# About:
+
+ClickableText looks different from Nri.Ui.Button in that it displays without margin or padding.
+ClickableText has the suave, traditional look of a "link"!
+
+For accessibility purposes, buttons that perform an action on the current page should be HTML `<button>`
+elements and are created here with `*Button` functions. Buttons that take the user to a new page should be
+HTML `<a>` elements and are created here with `*Link` functions.
+
+
+# `<button>` creators
+
+@docs button, link
+
+
+# Config
+
+@docs Size
+
+-}
+
+import Css
+import Css.Global
+import Html.Styled as Html exposing (..)
+import Html.Styled.Attributes as Attributes
+import Html.Styled.Events as Events
+import Nri.Ui
+import Nri.Ui.Colors.V1 as Colors
+import Nri.Ui.Fonts.V1
+import Nri.Ui.Icon.V4 as Icon exposing (IconType)
+
+
+{-| Sizes for the button
+-}
+type Size
+    = Small
+    | Medium
+    | Large
+
+
+{-| Config for the button
+-}
+type alias ButtonConfig msg =
+    { label : String
+    , size : Size
+    , icon : Maybe IconType
+    , onClick : msg
+    }
+
+
+{-| Creates a `<button>` element
+-}
+button : ButtonConfig msg -> Html msg
+button config =
+    let
+        fontSize =
+            sizeToPx config.size
+    in
+    Nri.Ui.styled Html.button
+        (dataDescriptor "button")
+        (clickableTextStyles fontSize)
+        [ Events.onClick config.onClick
+        ]
+        [ icon fontSize config.icon
+        , text config.label
+        ]
+
+
+{-| Config for the link
+-}
+type alias LinkConfig =
+    { label : String
+    , size : Size
+    , icon : Maybe IconType
+    , url : String
+    }
+
+
+{-| Creates a `<a>` element
+-}
+link : LinkConfig -> List (Attribute msg) -> Html msg
+link config additionalAttributes =
+    let
+        fontSize =
+            sizeToPx config.size
+    in
+    Nri.Ui.styled Html.a
+        (dataDescriptor "link")
+        (clickableTextStyles fontSize)
+        (Attributes.href config.url
+            :: additionalAttributes
+        )
+        [ icon fontSize config.icon
+        , text config.label
+        ]
+
+
+clickableTextStyles : Css.Px -> List Css.Style
+clickableTextStyles fontSize =
+    [ Css.cursor Css.pointer
+    , -- Specifying the font can and should go away after bootstrap is removed from application.css
+      Nri.Ui.Fonts.V1.baseFont
+    , Css.backgroundImage Css.none
+    , Css.textShadow Css.none
+    , Css.property "transition" "background-color 0.2s, color 0.2s, box-shadow 0.2s, border 0.2s, border-width 0s"
```

Comment:
🎨 would we want to use https://package.elm-lang.org/packages/rtfeldman/elm-css/latest/Css-Transitions ?

Reply:
Ooh! Didn't know about that, yes!